crates/stark-backend/src/keygen/mod.rs

```diff
@@ -195,11 +196,33 @@ impl<'a, SC: StarkGenericConfig> MultiStarkKeygenBuilder<'a, SC> {
             threshold: log_up_security_params.max_interaction_count,
         });
 
+        let pre_vk: MultiStarkVerifyingKey0<SC> = MultiStarkVerifyingKey0 {
+            per_air: pk_per_air.iter().map(|pk| pk.vk.clone()).collect(),
+            trace_height_constraints: trace_height_constraints.clone(),
+            log_up_pow_bits: log_up_security_params.log_up_pow_bits,
+        };
+        // To protect against weak Fiat-Shamir, we hash the "pre"-verifying key and include it in the
+        // final verifying key. This just needs to commit to the verifying key and does not need to be
+        // verified by the verifier, so we just use bincode to serialize it.
+        let vk_bytes = bitcode::serialize(&pre_vk).unwrap();
```

Comment:
Noting that if we change bitcode versions this may change.

Reply:
yea this is why I wanted bincode v2.0.0 (more stable), I'm also open to just fixing it to bincode v1.3.3 for more stability?

Reply:
switched to `bincode`

Reply:
I think it's ok since this is not really externally used and does not need to be reconstructable after keygen